modules/TwitterStreamingImporter/src/main/nbm/manifest.mf

```diff
@@ -1,4 +1,4 @@
-Manifest-Version: 1.3
+Manifest-Version: 1.4
```

Comment:
I don't think the manifest version should be ever different than `1.0` though. It's not the project version but the file format version.

Reply:
Oh stupid I am :D I'll change this